lib/pages/inventory.dart

```diff
@@ -1,8 +1,69 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_redux/flutter_redux.dart';
+import 'package:scibble/models/product.dart';
+import 'package:scibble/redux/inventory/actions.dart';
+import 'package:scibble/redux/store.dart';
+import 'package:scibble/theme/scibble_color.dart';
+import 'package:scibble/widgets/hamburger.dart';
+import 'package:scibble/widgets/scibble_app_bar.dart';
 
 class Inventory extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    return Text('Inventar');
+    return Scaffold(
```

Comment:
You don't need to create a Scaffold in this view. The wrapping components (top app bar, nav bar) is taken care of by `home.dart`.  You only need to return the `StoreConnector`.